.github/workflows/main.yml

```diff
@@ -46,7 +47,13 @@ jobs:
     - name: Build
       env:
         EXILED_REFERENCES: ./References
-      run: msbuild EXILED.sln -p:Configuration=release
+      run: |
+        msbuild EXILED.API -p:Configuration=release
```

Comment:
Shouldn't `EXILED` be lowercase?

Reply:
No, it finds the folder without any problems, as you can see from the successful CI.

Reply:
I know, because it's case insensitive, but I mean, we should name it `Exiled` to be more consistent.

Reply:
Uh, I didn't notice that there is NormalCase after.